pom.xml

```diff
@@ -439,7 +439,7 @@
                     <configuration>
                         <verbose>false</verbose>
                         <failOnViolation>true</failOnViolation>
-                        <includeTestSourceDirectory>true</includeTestSourceDirectory>
+                        <includeTestSourceDirectory>false</includeTestSourceDirectory>
```

Comment:
Why do we have to turn it off here and in the child pom.xml files?

Reply:
Updated the child pom.xml files to remove the redundant settings.